src/onboarding/steps.ts

```diff
@@ -297,7 +303,11 @@ export function _getStepInfo({ firstScreenInStep, navigator, dispatch, props }:
         next: () => {
           if (skipVerification) {
             dispatch(setHasSeenVerificationNux(true))
-            finishOnboarding(Screens.WalletHome)
+            // This seems like a bug where we don't go to CYA on
+            // skipping phone verification. Every other case navigates to CYA at
+            // the end of onboarding, including when skipping verification on
+            // the import flow. TODO(ACT-1114): make this consistent with other cases
+            finishOnboarding(Screens.TabHome)
```

Comment:
@jh2oman can you confirm that this was just missed in https://github.com/valora-inc/wallet/pull/3564/files#diff-480de4d7238ccb6e436b320c3a9d8eb69160c00e8a336578436545be0864ee52 and there wasn't a specific reason to skip CYA screen just for this case?

Reply:
I think it must have just been missed. I can't think of why we wouldn't want to show choose your adventure for this case in particular